xpdan/tests/test_formatters.py

```diff
@@ -0,0 +1,55 @@
+##############################################################################
+#
+# xpdan            by Billinge Group
+#                   Simon J. L. Billinge sb2896@columbia.edu
+#                   (c) 2016 trustees of Columbia University in the City of
+#                        New York.
+#                   All rights reserved
+#
+# File coded by:    Elizabeth Culbertson
+#
+# See AUTHORS.txt for a list of people who contributed.
+# See LICENSE.txt for license information.
+#
+##############################################################################
+import os
+
+import pytest
+from xpdan.formatters import (clean_template, get_filename_prefix, render_and_clean)
+import yaml
+
+def get_md_example(filename):
```

Comment:
Please use 4 space intent.

Reply:
Happy to fix this but I'm not seeing where this isn't done -- where do I not 4 space indent? (Still getting used to this IDE so I don't doubt I messed it up in a few places)